starship

```diff
@@ -97,9 +101,9 @@ if __name__ == "__main__":
                         client.AUTHENTICATION_TOKEN))
             sys.exit(1)
         else:
-            # Hack at the moment, but should check status code 200 against test endpoint
-            sys.stdout.write("You have successfully authenticated to the platform!\n")
-            sys.exit()
+            sys.stderr.write("Could not connect to server: {}, {}\n".format(
```

Comment:
Can we use f-strings here?

Reply:
Strawberry Fields has Python 3.6 as target for compatibility, so not yet.

Reply:
(Python 3.5+ currently.)

Reply:
Sorry I meant to say 3.5+!